stacks/api-stack.js

```diff
@@ -0,0 +1,16 @@
+import { Api } from '@serverless-stack/resources'
+
+/**
+ * @param {import('@serverless-stack/resources').StackContext} properties
+ */
+export function ApiStack({ stack }) {
+  const api = new Api(stack, 'api', {
```

Comment:
minor: this resource are going get auto named `w3up-api-api`

Reply:
I did not use default service name because `service` is already used in a bunch of places to name the ucanto remote service. Any other naming suggestions we could use?

Reply:
`http-gateway` is the actual aws resource that we's be creating so we could go with that